reddit/plugin_web.go

```diff
@@ -142,6 +142,10 @@ func HandleNew(w http.ResponseWriter, r *http.Request) interface{} {
 		Disabled:   false,
 	}
 
+	if ok, _ := CheckSubreddit(newElem.Subreddit); !ok {
```

Comment:
Is there a reason `CheckSubreddit` returns `(bool, error)` rather than simply `bool`, given that the only callsite only uses the first return value?

(If you do change `CheckSubreddit` to return a single value, perhaps call it `SubredditExists` or something along those lines to suggest that it returns a boolean.)